diff --git a/node_modules/jsonresume-theme-engineering/resume.hbs b/node_modules/jsonresume-theme-engineering/resume.hbs
index 3c8e1cc..79d4cd3 100644
--- a/node_modules/jsonresume-theme-engineering/resume.hbs
+++ b/node_modules/jsonresume-theme-engineering/resume.hbs
@@ -18,8 +18,10 @@
         {{> basics }}
         {{> skills }}
         {{> work }}
+        {{> projects }}
         {{> volunteer }}
         {{> education }}
+        {{> languages }}
         {{> awards }}
     </body>
 </html>
diff --git a/node_modules/jsonresume-theme-engineering/views/languages.hbs b/node_modules/jsonresume-theme-engineering/views/languages.hbs
new file mode 100644
index 0000000..1c7efa2
--- /dev/null
+++ b/node_modules/jsonresume-theme-engineering/views/languages.hbs
@@ -0,0 +1,9 @@
+{{#if resume.languages.length}}
+<section id="languages">
+    <h2>Languages</h2>
+    {{#each resume.languages}}
+    {{#> item title=language subtitle=fluency}}
+    {{/item}}
+    {{/each}}
+</section>
+{{/if}}
diff --git a/node_modules/jsonresume-theme-engineering/views/projects.hbs b/node_modules/jsonresume-theme-engineering/views/projects.hbs
new file mode 100644
index 0000000..9e70560
--- /dev/null
+++ b/node_modules/jsonresume-theme-engineering/views/projects.hbs
@@ -0,0 +1,16 @@
+{{#if resume.projects.length}}
+<section id="projects">
+    <h2>Projects</h2>
+    {{#each resume.projects}}
+        {{#> item title=name subtitle=description startDate=startDate endDate=endDate}}
+        {{#if highlights.length}}
+        <ul class="highlights">
+            {{#each highlights}}
+            <li>{{{.}}}</li>
+            {{/each}}
+        </ul>
+        {{/if}}
+        {{/item}}
+    {{/each}}
+</section>
+{{/if}}
